### PR TITLE
Implement new Profiles cover design

### PR DESCRIPTION
**Pull request by Tanza3D**
image
image
image
image

### CODE REVIEW

File: profiles/index.php [hunk 1]
```diff
@@ -300,7 +310,8 @@
                 <section class="osekai__panel">
                     <div id="AddEntryPanel" class="osekai__panel-header-with-buttons">
                         <div class="osekai__panel-hwb-left">
-                        <i class="fas fa-stream"></i><p><?php echo GetStringRaw("profiles", "profile.timeline.title"); ?></p>
+                            <i class="fas fa-stream"></i>
+                            <p><?php echo GetStringRaw("profiles", "profile.timeline.title"); ?></p>
```
Comment (EXtremeExploit) [inline profiles/index.php:314]:
```suggestion
                            <p><?= GetStringRaw("profiles", "profile.timeline.title"); ?></p>

```
same here

File: profiles/index.php [hunk 2]
```diff
@@ -181,46 +183,54 @@
...
+                                </div>
+                                <div class="profiles__cover-bottom_text">
+                                    <p><i class="fas fa-link"></i><a target="_blank" id="website"><?php echo GetStringRaw("general", "loading.longer"); ?></a></p>
+                                </div>
                             </div>
-                            <div class="profiles__cover-back-info_text">
-                                <p><i class="fab fa-twitter"></i><a target="_blank" id="twitter"><?php echo GetStringRaw("general", "loading.longer"); ?></a></p>
+                            <div class="profiles__cover-bottom-row">
+                                <div class="profiles__cover-bottom_text">
+                                    <p id="location"><i class="fas fa-map-marker-alt"></i> <?php echo GetStringRaw("general", "loading.longer"); ?></p>
```
Comment (EXtremeExploit) [inline profiles/index.php:218]: oh and all over this section

File: profiles/js/functions.js [hunk 3]
```diff
@@ -235,10 +235,21 @@ async function FillData(uid, mode, completeReload = true) {
...
         }
         if (oData.username != null) document.getElementById("name__sub").innerHTML = `
-    <img class="profiles__cover-country" src="https://osu.ppy.sh/images/flags/${oData.country_code}.png" id="country__flag"> ${oData.username} <div id="user__badges" class="profiles__user-badges"></div>
-    <a data-tippy-content="View profile on osu.ppy.sh" href="https://osu.ppy.sh/users/${oData.id}" target="_blank" class="profiles__cover-link"><i class="fas fa-external-link-alt"></i></a>`;
+        <div class="profiles__cover-info-name">
+            <img class="profiles__cover-country" src="https://osu.ppy.sh/images/flags/${oData.country_code}.png" id="country__flag">
+            <h1>${oData.username}</h1> 
+        </div>
+        <div id="user__badges" class="profiles__user-badges"></div>`;
+        document.getElementById("osu_link").href=  `https://osu.ppy.sh/users/${oData.id}`;
```
Comment (EXtremeExploit) [inline profiles/js/functions.js:243]: super scuffed formatting here
```suggestion
        document.getElementById("osu_link").href = `https://osu.ppy.sh/users/${oData.id}`;

```

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]

File: profiles/index.php (hunk 2 above)
Comment (Tanza3D) [reply profiles/index.php:218]: what's wrong with it?

Comment (Tanza3D) [reply profiles/index.php:218]: oh the <?php echos i gotcha

**Review by EXtremeExploit — APPROVED** [review]

**Pull request merged**